BitmarkRegistry/Services/FileCourierService.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  FileCourierService.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 7/3/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import Foundation
+import BitmarkSDK
+import Alamofire
+import MobileCoreServices
+
+class FileCourierServer {
+
+  static func updateFileToCourierServer(
+    assetId: String, encryptedFileURL: URL,
+    sender: Account, senderSessionData: SessionData,
+    receiverAccountNumber: String, receiverSessionData: SessionData) {
+
+    guard let jwt = KeychainStore.getJwtFromKeychain() else { return }
+    let assetFilename = encryptedFileURL.lastPathComponent
+
+    do {
+      let uploadURL = URL(string: Global.ServerURL.fileCourier + "/v2/files/" + assetId + "/" + sender.getAccountNumber())!
+
+      let access = "\(receiverAccountNumber):\(receiverSessionData.encryptedKey.hexEncodedString)"
+      let parameters: [String : String] = [
+        "data_key_alg" : senderSessionData.algorithm,
+        "enc_data_key" : senderSessionData.encryptedKey.hexEncodedString,
+        "orig_content_type" : "*",
```

Comment:
`orig_content_type` is mime type.

Reply:
Thanks for letting me know. 
before, somehow I tested to call the API without "data_key_alg", "enc_data_key", "orig_content_type" paramaters; and it was still success as I remembered.

I will test it again and update to another PR.